css/css-color/xyz-004-ref.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Color 4: CSS Color 4: xyz</title>
+<style>
+    .test { background-color: lab(100% -431.0345 172.4138); width: 12em; height: 12em; } /* color(xyz 0 1 0) converted to Lab */
```

Comment:
Because the X Y and Z primaries are super-saturated (not physically realizable), all colors apart from black have all three components non-zero.
The resulting a and b values are thus large (outside the range of many integer or fixed point encodings, which are often limited to ±128.0) and will clearly be gamut-mapped on any physical display.
Besides testing that lab() and xyz() syntaxes are supported, what is the intent of this test?

Reply:
Same here, no intent, just nice numbers, sorry :(.